src/extension/EmulatorManager.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as nls from "vscode-nls";
+import { QuickPickOptions, window } from "vscode";
+nls.config({ messageFormat: nls.MessageFormat.bundle, bundleFormat: nls.BundleFormat.standalone })();
+const localize = nls.loadMessageBundle();
+
+export interface IEmulator {
+    name?: string;
+    id: string;
+}
+
+export abstract class EmulatorManager {
```

Comment:
If this class will be used as multiplatform one I think `VirtualDeviceManager` will look better